mrp_partial_production/models/mrp.py

```diff
@@ -1,40 +1,17 @@
 # coding: utf-8
-###########################################################################
-#    Module Writen to OpenERP, Open Source Management Solution
-#
-#    Copyright (c) 2013 Vauxoo - http://www.vauxoo.com
-#    All Rights Reserved.
-#    info@vauxoo.com
-############################################################################
-#    Coded by: Jose Morales <jose@vauxoo.com>
-#    Planned by: Nhomar Hernandez <nhomar@vauxoo.com>
-#    Audited by: Jose Morales <jose@vauxoo.com>
-############################################################################
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-from openerp import models, fields, api, _
-from openerp.exceptions import Warning as UserError
+# Copyright 2017 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from collections import defaultdict
+from odoo import api, fields, models
 
 
 class MrpProduction(models.Model):
 
     _inherit = 'mrp.production'
 
     @api.multi
-    @api.depends('state', 'move_lines', 'move_lines2')
+    @api.depends('state', 'move_raw_ids')
     def _compute_qty_to_produce(self):
```

Comment:
could you check this method?
adapt with filtered

Reply:
The change is in function [test_ready](https://github.com/Vauxoo/addons-vauxoo/pull/1227/commits/37a25f3f6d1cf5a198d2781c1af4b2b8aebc0241)

Reply:
Function [test_ready](https://github.com/Vauxoo/addons-vauxoo/pull/1227/commits/7825bb6baec29767c20a0811e877221a255debbf) was remove because in saas-17 does not exist.

Reply:
👍

Reply:
is not necessary to this module the behavior of that method?

Reply:
No, that method existed until v.9.0 and was used to activate workflow